src/components/triSwitch.vue

```diff
@@ -1,9 +1,20 @@
 <template>
-	<svg :width="wheelDimensions.width" :height="wheelDimensions.height/2" @click="changeOption">
-		<g :transform="'translate(' + wheelDimensions.width/2 + ', ' + wheelDimensions.height/4 + ')'">
-			<g class="indicator" v-for="noun in localisations" v-bind:style="{transform: 'rotate(' + noun.rotation + 'deg)', visibility: noun.visibility}">
+	<svg :width="dimensions.width" :height="dimensions.height/2" 
+		@click="changeOption">
+
+		<g :transform="'translate(' 
+			+ dimensions.width/2 + ', ' + dimensions.height/4 + ')'">
```

Comment:
It is mostly a styling thing for myself - The <g> tag should be an indent of the <svg>, but the svg items are also indented since they exceed the line length of 80. This is a good opportunity to discuss html indentation conventions for the project.

We can opt for what I am doing:
```html
<html lang="en"... lot of other things
    style="red">
    
        <div class=...>
        </div>
</html>
```
or go with something like:
```html
<html lang="en"... lot of other things
    style="red">
        <div class=...>
        </div>
</html>
```